actors/builtin/miner/miner_test.go

```diff
@@ -3617,6 +3908,41 @@ func (h *actorHarness) setProofType(proof abi.RegisteredSealProof) {
 	require.NoError(h.t, err)
 }
 
+func (h *actorHarness) setBeneficiary(rt *mock.Runtime, beneficiaryAddr addr.Address, beneficiary miner.BeneficiaryInfo) {
+	beneficiaryIdAddr, found := rt.GetIdAddr(beneficiaryAddr)
+	require.Equal(h.t, found, true)
+	h.beneficiary = beneficiaryIdAddr
+	h.beneficiaryInfo = miner.BeneficiaryInfo{
+		Quota:      beneficiary.Quota,
+		Expiration: beneficiary.Expiration,
+	}
+}
+
+func (h *actorHarness) changeOwnerToNewBeneficiary(rt *mock.Runtime, beneficiaryAddr addr.Address, beneficiary miner.BeneficiaryInfo) {
```

Comment:
This method name is a bit confusing. How about "proposeAndApproveInitialBeneficiary"?